configs/terraform/environments/dev/artifact-registry/terraform.tfvars

```
@@ -1,6 +1,9 @@
-gcp_region                     = "europe-west4"
-artifact_registry_module       = "cap-operator"
-artifact_registry_owner        = "neighbors"
-artifact_registry_type         = "development"
-immutable_artifact_registry    = false
-artifact_registry_multi_region = true
\ No newline at end of file
+gcp_region = "europe-west4"
+artifact_registry_list = [
+  {
+    name                   = "modules-internal"
+    owner                  = "neighbors"
+    type                   = "development"
+    reader_serviceaccounts = ["klm-controller-manager@sap-ti-dx-kyma-mps-dev.iam.gserviceaccount.com", "klm-controller-manager@sap-ti-dx-kyma-mps-stage.iam.gserviceaccount.com", "klm-controller-manager@sap-ti-dx-kyma-mps-prod.iam.gserviceaccount.com"]
+  },
+]
```

```suggestion
gcp_region = "europe-west4"
artifact_registries = {
  modules-internal = {
    name                   = "modules-internal"
    owner                  = "neighbors"
    type                   = "development"
    reader_serviceaccounts = ["klm-controller-manager@sap-ti-dx-kyma-mps-dev.iam.gserviceaccount.com", "klm-controller-manager@sap-ti-dx-kyma-mps-stage.iam.gserviceaccount.com", "klm-controller-manager@sap-ti-dx-kyma-mps-prod.iam.gserviceaccount.com"]
  },
}
```
Alternatively, can we use a map of objects and pass a raw variable to `for_each`

```
for_each = var.artifact_registries
```